libnmstate/netapplier.py

```diff
@@ -150,6 +152,13 @@ def _apply_ifaces_state(
         raise NmstatePermissionError('Error creating a check point')
     except nm.checkpoint.NMCheckPointCreationError:
         raise NmstateConflictError('Error creating a check point')
+    except NmstateError:
+        # Assume rollback occured, revert IPv6 stack state.
+        # Checkpoint rollback is async, there is a need to wait for it to
+        # finish before proceeding with other actions.
+        time.sleep(5)
```

Comment:
Maybe we need a `TODO` tag here to indicate we should replace sleep with rollback wait function once we have.

Reply:
I replaced them after your findings of the test fail cause.
Basically, I used your #449 idea with some changes.

Reply:
Done